strategies.py

```diff
@@ -175,14 +173,17 @@ def pick_move(self):
 
         Highest N is most robust indicator. In the early stage of the game, pick
         a move weighted by visit count; later on, pick the absolute max.'''
-        if self.root.position.n >= self.temp_threshold:
-            fcoord = np.argmax(self.root.child_N)
-        else:
-            cdf = self.root.child_N.cumsum()
+        if (not self.two_player_mode and
+                self.root.position.n < self.temp_threshold):
+            cdf = self.root.children_as_pi(squash=True).cumsum()
```

Comment:
This is a change in behavior (to select from the squashed distribution), This seems like what we SHOULD have been doing all along but maybe I misunderstand the code

Reply:
I'm not sure I'm fully on board with the squash thing; it always seemed a bit voodoo to me.

Reply:
`temp_threshold` (the move at which we switch from softpick to argmax) is set to be -1 if `two_player_mode` is true.  So this doesn't change behavior.

the squash thing is a little weird.

Reply:
(this means the check for `self.two_player_mode` is not needed and can be removed)